frontend/ui/data/koptoptions.lua

```diff
@@ -129,6 +149,8 @@ local KoptOptions = {
                 end,
                 labels = {S.AUTO, S.LEFT, S.CENTER, S.RIGHT, S.JUSTIFY},
                 name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[In reflow mode, sets the lines alignment.
```

Comment:
I think you meant "lines' alignment" but I'm not sure if that's any clearer than just alignment?

Reply:
`text alignment` then ? (avoid wondering if we align words (on a line) or the line itself)